.github/workflows/main.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - run: yarn
-    - run: yarn lint
+    - run: npm install
```

Comment:
Because yarn does not run prepare in the dependencies, libp2p-interfaces branch does not have types